Assets/Scripts/Projects/ProjectDataHandler.cs

```diff
@@ -1,7 +1,11 @@
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.InteropServices;
+using JetBrains.Annotations;
```

Comment:
wat is dit?

Reply:
Dit is voor de annotatie `[UsedImplicitly]`. Dit vertelt Rider dat die hetgeen daaronder niet moet markeren als ongebruikt; dan is het duidelijker dat de hetgeen daaronder geen dode code is. Doordat we pragma's gebruiken tonen dingen soms als ongebruikt en op deze manier kan ik ook gebruik maken van de statische analyse van Rider om te zien waar problemen in de code zitten.